test/nodes/test_chunker.py

```
@@ -0,0 +1,687 @@
+        iglobal = MagicMock()
+        iglobal.strategy = strategy
+        inst.IGlobal = iglobal
+
+        mock_instance = MagicMock()
+        inst.instance = mock_instance
+
+        return inst, mock_instance
+
+    def test_raises_runtime_error_when_strategy_is_none(self):
+        """Raise RuntimeError when strategy is None."""
+        saved, names, IInstance, RC = self._load_iinstance()
```

Ruff (0.15.9)

[warning] Unpacked variable `RC` is never used

Prefix it with an underscore or any other dummy variable pattern

(RUF059)